lib/src/kbasesearchengine/main/SearchMethods.java

```diff
@@ -267,6 +268,12 @@ public SearchTypesOutput searchTypes(SearchTypesInput params, String user) throw
     public SearchObjectsOutput searchObjects(SearchObjectsInput params, String user) 
             throws Exception {
         long t1 = System.currentTimeMillis();
+
+        // validate input
+        if(params.getObjectTypes() == null) {
```

Comment:
space after if

Reply:
Fixed. I have intellij doing this for me now. Should not happen again.